packages/components/package.json

```diff
@@ -148,6 +149,8 @@
     "jest-cli": "^29.7.0",
     "postcss-import": "16.1.1",
     "puppeteer": "^24.3.0",
+    "react": "^18.3.1",
```

Comment:
not sure about including react as a dependency. We are committed to being agnostic to the framework